### PR TITLE
Add github action to run TKG test

### PR DESCRIPTION
**Pull request by renfeiw**
- run tkg test with comment "run tkg-test [TEST_OPTIONS]" in PR

### CODE REVIEW

**Review by smlambert — APPROVED** [review]
thanks for the demo in the Tuesday call @renfeiw !

File: .github/workflows/testTKG.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    types: [created]
+jobs:
+  testTKG:
+    runs-on: ubuntu-latest
+    if: startsWith(github.event.comment.body, 'run tkg-test')
+    steps:
+      - uses: xt0rted/pull-request-comment-branch@v1
+        id: comment-branch
+
+      - uses: actions/checkout@v3
```
Comment (karianna) [inline .github/workflows/testTKG.yml:13]: These should be at the has commit version with a comment on the version tag

**Review by karianna — CHANGES_REQUESTED** [review]

File: .github/workflows/testTKG.yml (hunk 1 above)
Comment (renfeiw) [reply .github/workflows/testTKG.yml]: Updated, thanks!

**Review by llxia — APPROVED** [review]
LGTM

**Review by karianna — APPROVED** [review]

**Pull request merged**